data_steward/tools/import_rdr_omop.py

```diff
@@ -86,6 +86,8 @@ def create_rdr_tables(client, rdr_dataset, bucket):
     for table, schema in schema_dict.items():
         schema_list = bq.get_table_schema(table, schema)
         table_id = f'{project}.{rdr_dataset}.{table}'
+        table = bigquery.Table(table_id, schema=schema_list)
+        table = client.create_table(table)
```

Comment:
I'm not crazy about the way this loop is structured now.  Proposed lines 89 and 90 are a functional duplicate of current lines 126 and 127. With this addition, we create each table, then try to write truncate the table on load, and if the file doesn't exist, try to create the table again in line 129 (in the patch), which will probably raise a Table Already Exists error because these lines executed.

I think we should just remove the conditional logic at line 123/125 (current line/patched line) and move the entire conditional block over.
```
            if table in AOU_REQUIRED:
``` 
to achieve the same effect and ensure we are only creating each table once.  Instead of adding these two lines.

The current patched approach is to create the table, then recreate the table at either file load or file load failure.
This changes the original approach of "create the table once, either at file load or as an empty table".